index.html

```diff
@@ -6,8 +6,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
+    <script src="login.js"></script>
 </head>
+
 <body>
-    <h1>comproDLS</h1>
+    <main>
+        <div class="container">
+            <div class="container__left">
+                <div class="info-area">
+                    <p class="info-area__h1">
+                        comproDLS<span>TM</span>
+                    </p>
+                    <p class="info-area__h2">
+                        OnBoarding
+                    </p>
+                    <hr id="info-area__hr">
```

Comment:
Here info-area__hr does not serve any special purpose (ie for identification or for styling). So you can use your responsive utility classes instead (web-visi, mob-visi) which ever is applicable, to clean some clutter.